runtime-light/state/component-state.cpp

```diff
@@ -54,6 +56,40 @@ void ComponentState::parse_runtime_config_arg(std::string_view value_view) noexc
   }
 }
 
+void ComponentState::parse_command_line_arg(std::string_view value_view) noexcept {
+  if (value_view.empty()) [[unlikely]] {
+    php_warning("command line argument is empty");
+    return;
+  }
+
+  if (!command_line_argv.empty()) [[unlikely]] {
+    php_warning("command line argument support no more one usage");
+    return;
+  }
+
+  const auto& main_file_view{kphp::compiler_interface::get_main_file_name()};
+  command_line_argv.push_back(string(main_file_view.data(), main_file_view.size()));
+
+  bool in_quote{};
+  string current_arg{};
```

Comment:
It seems like an error. You need to set special reference counter for each reference counted object